requirements.txt

```diff
@@ -7,3 +7,5 @@ tabulate
 testslide>=2.7.0
 typing_extensions
 typing_inspect
+tomli
+tomli-w
```

Comment:
This file should be kept sorted to make it easy to spot missing dependencies and nearly impossible to insert duplicates.